### PR TITLE
Create Wireless Power Transfer.md

### PR DESCRIPTION
**Pull request by troydunn03**

### CODE REVIEW

**Review by JesseTNRoberts — COMMENTED** [review]
I need to know what this is supposed to do. Get rid of the discussion on how you plan to do it in the intended function section.

Second, are you certain that wireless power transfer as a means to actually power the car is a good idea? What sort of transmitter are you imagining? As the car moves, you will lose coupling unless this is some sort of a large mat (which could be done). However, I don't see anything like that in the design below. 

Your BOM does not detail what the items are.

**Review by JesseTNRoberts — COMMENTED** [review]
This has been significantly updated! One issue is, it does not consider my question regarding transmitter. You have simulated this using LT spice (very good). But you assume a certain coupling co-efficient. What will happen when the car moves? (the coupling co-efficient is going to change)

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+## Function 
+
+The purpose of this subsystem is to demonstrate wireless charging, energy, and power concepts through user interaction. The voltage will be adjustable by the user to control different charge rates, allowing them to analyze different drive times of the car depending on the power supply and time of charge. This will allow the user to learn how power and energy behave through the use of wireless charging. 
+
+## Constraints
+
+|Description    |Constraint                                                                         |Source              |
+|---------------|-----------------------------------------------------------------------------------|--------------------|
+|Voltage        |The system shall operate at less than 50 V                                         |Conceptual Design   |
+|Voltage        |The output shall not exceed 4.2 V                                                  |[1]                 |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: give the source not a reference here.

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+## Function 
+
+The purpose of this subsystem is to demonstrate wireless charging, energy, and power concepts through user interaction. The voltage will be adjustable by the user to control different charge rates, allowing them to analyze different drive times of the car depending on the power supply and time of charge. This will allow the user to learn how power and energy behave through the use of wireless charging. 
+
+## Constraints
+
+|Description    |Constraint                                                                         |Source              |
+|---------------|-----------------------------------------------------------------------------------|--------------------|
+|Voltage        |The system shall operate at less than 50 V                                         |Conceptual Design   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: Why?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+|Voltage        |The output shall not exceed 4.2 V                                                  |[1]                 |
+|Frequency      |The system shall operate between 110 kHz and 205 kHz                               |LTC2140 Datasheet   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: Why?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+|Ethical        |The system shall have the capability of providing more than 75% effieciency        |Conceptual Design   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: Why?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 5]
```diff
@@ -6,17 +6,17 @@ The purpose of this subsystem is to demonstrate wireless charging, energy, and p
...
-|---------------|-----------------------------------------------------------------------------------|--------------------|
-|Voltage        |The system shall operate at less than 50 V                                         |Conceptual Design   |
-|Voltage        |The output shall not exceed 4.2 V                                                  |[1]                 |
-|Frequency      |The system shall operate between 110 kHz and 205 kHz                               |LTC2140 Datasheet   |
-|Ethical        |The system shall have the capability of providing more than 75% effieciency        |Conceptual Design   |
+|Description    |Constraint                                                                                                            |Source              |
+|---------------|----------------------------------------------------------------------------------------------------------------------|--------------------|
+|Voltage        |The system shall operate at less than 50 V to ensure safety in an educational setting.                                |Conceptual Design   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: How do you know that is safe?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 6]
```diff
@@ -6,17 +6,17 @@ The purpose of this subsystem is to demonstrate wireless charging, energy, and p
...
+|Voltage        |The output shall not exceed 4.2 V to ensure safe battery charging levels.                                             |LTC4120 Datasheet   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: How do you know that is safe?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 7]
```diff
@@ -6,17 +6,17 @@ The purpose of this subsystem is to demonstrate wireless charging, energy, and p
...
-|Voltage        |The output shall not exceed 4.2 V                                                  |[1]                 |
-|Frequency      |The system shall operate between 110 kHz and 205 kHz                               |LTC2140 Datasheet   |
-|Ethical        |The system shall have the capability of providing more than 75% effieciency        |Conceptual Design   |
+|Description    |Constraint                                                                                                            |Source              |
+|---------------|----------------------------------------------------------------------------------------------------------------------|--------------------|
+|Voltage        |The system shall operate at less than 50 V to ensure safety in an educational setting.                                |Conceptual Design   |
+|Voltage        |The output shall not exceed 4.2 V to ensure safe battery charging levels.                                             |LTC4120 Datasheet   |    
+|Frequency      |The system shall operate between 110 kHz and 205 kHz to maximize efficiency between the transmitter and receiver.     | LTC2140 Datasheet  |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: How do you know that maximizes efficiency?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 8]
```diff
@@ -6,17 +6,17 @@ The purpose of this subsystem is to demonstrate wireless charging, energy, and p
...
+|Ethical        |The system shall provide more than 75% efficiency to decrease energy waste in the system.                             |Conceptual Design   |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: How are you going to measure this? And how is this ethical in nature?

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 9]
```diff
@@ -0,0 +1,244 @@
...
+
+## Function 
+
+The purpose of this subsystem is to demonstrate wireless charging, energy, and power concepts through user interaction. The voltage will be adjustable by the user to control different charge rates, allowing them to analyze different drive times of the car depending on the power supply and time of charge. This will allow the user to learn how power and energy behave through the use of wireless charging. 
+
+## Constraints
+
+|Description    |Constraint                                                                                                            |Source                      |
+|---------------|----------------------------------------------------------------------------------------------------------------------|----------------------------|
+|Standard       |The system shall operate at less than 50 V to ensure safety in an educational setting.                                |OSHA 1926.403(i)(2)(i)      |
```
Comment (JesseTNRoberts) [inline Documentation/Signoff/Wireless Power Transfer.md]: I see nothing about 50 volts in that standard.

Comment (troydunn03) [reply Documentation/Signoff/Wireless Power Transfer.md]: I'm confused on what you mean. I know that osha 1926.403(i)(2)(i) states the following: "Except as required or permitted elsewhere in this subpart, live parts of electric equipment operating at 50 volts or more shall be guarded against accidental contact by cabinets or other forms of enclosures, or by any of the following means:". I have it references but maybe I'm misunderstanding what you're saying.

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 10]
```diff
@@ -0,0 +1,149 @@
...
+|2.2uF Capacitor                      |Digikey        |1        |$0.17      |$0.17       |
+|15uH Inductor                        |Digikey        |1        |$1.19      |$1.19       |
+|3.01kohm Resistor                    |Digikey        |1        |$0.10      |$0.10       |
+|1Mohm Resistor                       |Digikey        |1        |$0.10      |$0.10       |
+|10kohm Resistor                      |Digikey        |1        |$0.10      |$0.10       |
+|1.35Mohm Resistor                    |Digikey        |1        |$0.66      |$0.66       |
+|22uF Capacitor                       |Digikey        |1        |$0.33      |$0.33       |
+|9V Lithium-ion Battery               |Amazon         |1        |$18.99     |$18.99      |
+|9V Lithium-ion Battery Connector     |Amazon         |1        |$4.99   v  |$4.99       |
+|                                     |               |         |Total:     |$71.27      |
```
Comment (micahr234) [inline Documentation/Signoff/Wireless Power Transfer.md]: Where are part numbers? Just specifying the component rating does not fully specify the component.

File: Documentation/Signoff/Wireless Power Transfer.md [hunk 11]
```diff
@@ -0,0 +1,149 @@
...
+
+$$F_{CS,CP} = \frac{1}{2*\pi*\sqrt{\left( C_{2P} + C_{2S} \right)*L_{R}}}$$
+
+$$F_{CS,CP} = 129.44\ kHz$$
+
+<img alt="LTC4120 Output Voltage and Current" src="https://github.com/user-attachments/assets/bbf0b336-671c-49f4-bec8-f265616cffb9">
+
+Figure 9. Output Voltage and Current
+
+The figure above shows the current and voltage outputs for the battery charger. The battery being charged will be a 9V lithium-ion battery which is capable of being charged at 4.2 V. From the simulation, the maximum voltage is 4.2V and the maximum current is 500mA. The LTC4120 is the optimal chip for charging batteries as it ensures a certain voltage and current output providing extra safety features.
```
Comment (micahr234) [inline Documentation/Signoff/Wireless Power Transfer.md]: Wait, a 9V battery will charge at 4.2volts? That does not sound right.

File: Documentation/Signoff/Wireless Power Transfer.md (hunk 9 above)
Comment (micahr234) [reply Documentation/Signoff/Wireless Power Transfer.md]: You must have the wrong OSHA standard number.

**Review by micahr234 — APPROVED** [review]

**Pull request merged**